### PR TITLE
replace dashboard tables to pie charts and re-order

### PR DESCRIPTION
**Pull request by GeneralSnuggles**
changed:
Script Execution Status
Most Frequently Executed Scripts
to be pie graphs and moved them up

need to decide regarding:
Timeline of Scripts Executed on Each Endpoint

and add:
Time since last Scan

want to check regarding clicking on segments in the chart to pull a list of the selected scripts

### CODE REVIEW

File: .env.swp
Comment (maziland) [inline .env.swp:1]: Remove this file, it is not needed

File: views/dashboard.ejs [hunk 1]
```diff
@@ -9,74 +9,116 @@
...
+            };
+    
+            // Render the Script Chart
+            const scriptCtx = document.getElementById('script-chart').getContext('2d');
+            new Chart(scriptCtx, {
+                type: 'pie',
+                data: scriptData,
+            });
+        });
+    </script>
```
Comment (maziland) [inline views/dashboard.ejs]: Create a file in `public/js` and call it `dashboard.js`, and insert a link in the html to load it.
It is hard to manage scripts inside html or ejs files

**Review by maziland — CHANGES_REQUESTED** [review]

File: views/dashboard.ejs (hunk 1 above)
Comment (GeneralSnuggles) [reply views/dashboard.ejs]: Done

**Review by maziland — APPROVED** [review]
Looks good.
I think you should add the option to make the table headers clickable, and to sort by ascending or descending.
Should be an easy javascript

**Pull request merged**